config.yml

```diff
@@ -136,7 +136,7 @@ genesis:
         max_delegated_gateways: "7"
       applicationList:
         - address: pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4
-          delegatee_gateway_addresses: []
+          delegatee_gateway_addresses: [ ]
```

Comment:
Can you double check that [1] is clean or needs any other edits please?

[1] https://dev.poktroll.com/develop/developer_guide/adding_params